crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -2361,3 +2361,67 @@ ttccccC'''cC'-'Cc'''1'1cctt
      ttttttttttttttttt t
        tt   ttt ttt t
 ENDMAP
+
+###############################################################################
+# Growth, Death and Undeath, by byrel
+# 1/11 chance of an undead lernaean hydra
+NAME:    byrel_swamp_growth_death_and_undeath
+TAGS:    no_monster_gen
+PLACE:   Swamp:$
+ORIENT:  float
+TAGS:    no_rotate
+MONS:    swamp dragon / swamp drake / hydra
+MONS:	 freezing wraith / flayed ghost / phantasmal warrior / ghoul
```

Comment:
Whitespace is off here.

Reply:
> Cool idea. I'm not much of an expert on vault design / balance, so I'll leave that for gammafunk, but I have a couple internal-cleanup things to suggest.

All right, I made those changes. I doubt gammafunk is interested in reviewing this any further; is there anyone else on the dev team that typically reviews vault game design stuff?